Miracast/include/MiracastP2P.h

```diff
@@ -53,8 +53,11 @@ typedef enum p2p_events_e
 }
 P2P_EVENTS;
 
+#define ENABLE_DELAY_FOR_MIRACAST_P2P_SCAN
+
 #define MIRACAST_DFLT_NAME "Miracast-Generic"
 #define MIRACAST_DFLT_CFG_METHOD "pbc"
+#define MIRACAST_DFLT_P2P_SCAN_INTERVAL_MSEC "2000"
```

Comment:
remove this

Reply:
Addressed it as part of the https://github.com/rdkcentral/rdkservices/pull/4667/commits/740e0e7e079093a2cfc24217fcb8291944223489